### ISSUE TITLE
Current Viewer Tier State

### ISSUE THREAD

**Issue by wlongabaugh** [ImagingDataCommons/Viewers#84]
@fedorov @pieper cc: @madelyngreyes Current state of the tiers:

Dev: V 4.9.13 Build 137
Test: V 4.9.7, Build 123
Production: V 4.9.5, Build 121

When wanting to push from dev to test, or test to prod, please submit an issue here, and please let me know via Slack or email. If you wish to annotate with e.g. the major feature(s) of the version from IDC's perspective, this might be a good way to track it.

**Comment by madelyngreyes** [comment]
@fedorov @pieper hello, the viewer on the IDC test tier, doesn't work for segmentation. I believe it needs to updated with the IDC viewer version that's on the dev tier.

**Comment by pieper** [comment]
Thanks for checking @madelyngreyes  - Bill handles that side of things and he likes to get pinged on Slack when a new version needs to be deployed.

**Comment by fedorov** [comment]
@madelyngreyes can you provide the example where it does not work? This must be a regression, since I believe it works in production, and test (at least according to Bill note on top) is ahead of production. I am not sure upgrading will resolve the issue.

**Comment by madelyngreyes** [comment]
From the homepage, the selection of a CT option.
https://user-images.githubusercontent.com/16824894/118197986-0ae91c80-b41e-11eb-956d-283909605c09.mov

**Comment by fedorov** [comment]
You need to drag SEG into the viewport.

**Comment by madelyngreyes** [comment]
Yes, was able to get it working. Thank you for the clarification.

**Comment by wlongabaugh** [comment]
OK, current status is that one or two pull requests will be merged with dev in the near future, then I will get a notice here that we are ready to pull to test.

**Comment by Punzo** [comment]
@wlongabaugh I just merged into master version 4.9.17. Could you please update **dev** and **testing** portals (if @fedorov agrees)? @pieper could you please also update **sandbox**?

**Comment by fedorov** [comment]
I agree

**Comment by fedorov** [comment]
@wlongabaugh test tier viewer is working fine based on @madelyngreyes testing. Can we update the production tier?

**Comment by wlongabaugh** [comment]
@fedorov @pieper @madelyngreyes Released to production, backed by proxy with new scaling configuration: 
Dev: V 4.9.17 Build 141
Test: V 4.9.17, Build 143 (May 19)
Production: V 4.9.17, Build 145 (May 24)

**Comment by fedorov** [comment]
It feels so refreshing not to have to do hard reload and clear cache!

**Comment by fedorov** [comment]
@wlongabaugh the fork has been updated, and it it would be great to update test tier whenever you get a chance.

**Comment by Punzo** [comment]
Igor updated the fork to version 4.9,19, see https://github.com/ImagingDataCommons/Viewers/pull/91

**Comment by wlongabaugh** [comment]
@fedorov @pieper @madelyngreyes  Viewer updated in test:
 
Dev: V 4.9.20 Build 147 (June 3)
Test: V 4.9.20, Build 149 (June 3)
Production: V 4.9.17, Build 145 (May 24)

**Comment by fedorov** [comment]
Thanks! Just in time for @madelyngreyes testing in preparation to v2 release.

**Comment by wlongabaugh** [comment]
The prod version was old and thus I think it still had a sticky cache. It is now requesting V2 content.

**Comment by wlongabaugh** [comment]
@fedorov @pieper Has 4.10.1 been approved to push out to test and (next week) prod?
Dev: V 4.10.1 Build 167 (Sept 13)
Test: V 4.9.20 Build 149
Production: V 4.9.20, Build 161

**Comment by pieper** [comment]
I have not tested it, but the changes were small so I'd suggest putting it in the testing tier and letting people know they should kick the tires.

**Comment by wlongabaugh** [comment]
Fine with me. We can always hold prod at 4.9.20 if anything crops up.

Bill

**Comment by wlongabaugh** [comment]
And now we are sitting here:

Dev: V 4.10.1 Build 167 (Sept 13)
Test: V 4.10.1 Build 169 (Sept 20)
Production: V 4.9.20, Build 161

**Comment by wlongabaugh** [comment]
And now we are sitting here:

Dev: V 4.11.2 Build 176 (Oct 1)
Test: V 4.10.1 Build 169 (Sept 20)
Production: V 4.10.1, Build 174 (Sept 27)
